### DISCUSSION TITLE
Parameter does not match function name in dv8 showcase

### DISCUSSION THREAD

**Discussion by rnkazman** [sailuh/kaiaulu#335] [Q&A] [answered]
I got an error running this line:
filter_commit_size <- get_filter_commit_size(conf)

There is no parameter called filter_commit_size.  There is one called remove_filepaths_on_commit_size_greather_than
which turned out to be the one that I needed.  Also it would be good to have some advice/heuristics for reasonable settings for this parameter.

**Comment by carlosparadis** [comment, marked as answer]
Thank you for posting it here. Turns out that when we merged the new config.R functions, they were not 1 to 1. Will give some thought on defaults. Maybe a notebook on filtering could be helpful.
